hack/jenkins/windows_integration_setup.ps1

```diff
@@ -12,9 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-$test_root="$env:HOMEDRIVE$env:HOMEPATH\minikube-integration"
-$test_home="$test_root\$env:COMMIT"
+$test_home="$env:HOMEDRIVE$env:HOMEPATH\minikube-integration"
 $env:KUBECONFIG="$test_home\kubeconfig"
 $env:MINIKUBE_HOME="$test_home\.minikube"
 
+# delete in case previous test was unexpectedly ended and teardown wasn't run
+rm -r $test_home
```

Comment:
should this be -rf ? so it does't fail ?

Reply:
Added